### PR TITLE
Prepare release v0.1.1

### PR DESCRIPTION
**Pull request by LauraVilladakommit**
## Description

Release v.0.1.1 with the README updated.

### CODE REVIEW

**Review by jcamilocp — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,3 +6,7 @@
 - Set up tailwind and i18next libraries
 - Set up first version of integration and unit tests
 - Added first version of the CI
+
+## 0.1.1 (03.02.2023)
+
+- Update the README links to the support documents
```
Comment (miguelnietoa) [inline CHANGELOG.md:12]: This should be placed first

**Review by miguelnietoa — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jcamilocp) [reply CHANGELOG.md:12]: Right, we're going to correct it

Comment (miguelnietoa) [reply CHANGELOG.md:12]: Also, please update the text in the LICENSE section of the README. This project has Apache 2.0 license!

**Review by miguelnietoa — APPROVED** [review]
🚀

**Review by jcamilocp — APPROVED** [review]

**Pull request merged**